spec/system/task_spec.rb

```diff
@@ -3,34 +3,34 @@
 RSpec.describe 'Task', type: :system do
   describe 'Task一覧' do
     context '正常系' do
+      let(:project) { create(:project) }
+      let(:task) { create(:task, project_id: project.id) }
```

Comment:
associationの定義をFactory側に記載することで、letの記載が少なくなると思います！
```suggestion
      let(:task) { create(:task) }
```

Reply:
3e0f5713ec38f35e5005b4222d69c218fa80f71d で修正しました。

Reply:
https://github.com/nioi-sugoi/rspec_app_exam/pull/1#discussion_r470150883 で回答しました。